packages/components/src/avatar/avatar.tsx

```diff
@@ -130,34 +123,25 @@ const Indicator = styled(Stack, {
         backgroundColor: '$neutral-40',
       },
     },
-
-    shape: {
-      circle: {},
-      rounded: {
-        borderRadius: 16,
-      },
-    },
   } as const,
 })
 
 const Fallback = styled(Text, {
   name: 'AvatarFallback',
 })
 
-type BaseProps = GetProps<typeof Base>
-
 interface Props {
   src: string
-  size: NonNullable<BaseProps['size']>
+  size: 80 | 56 | 48 | 32 | 28 | 24 | 20 | 16
   indicator?: 'online' | 'offline'
   shape?: 'circle' | 'rounded'
-  withOutline?: boolean
+  outline?: boolean
 }
 
 type ImageLoadingStatus = 'idle' | 'loading' | 'loaded' | 'error'
 
 const Avatar = (props: Props) => {
-  const { src, size, shape = 'circle', withOutline, indicator } = props
+  const { src, size, shape = 'circle', outline = false, indicator } = props
```

Comment:
Can we add default value to indicator as well since it's optional prop?

Reply:
Default values are necessary when it affects styling. However, when the indicator is not defined, it should not appear. 

But yeah, we can add `none` variant, which does nothing.